lib/vanagon/platform/windows.rb

```diff
@@ -57,6 +57,16 @@ def generate_msi_packaging_artifacts(workdir, name, binding)
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/service.component.wxs.erb"), File.join(workdir, "wix", "service.#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+
+        # Project Specific Wix Files (ERB Templates)
+        ["customactions","properties","sequences"].each do |wix_filename|
```

Comment:
So, from what I can tell, this will fail if the file doesn't exist. That implies we always will require each project to provide these files. Have you tested that scenario?